miniwallet/config.ts

```diff
@@ -0,0 +1,146 @@
+import { ethers } from 'ethers'
+import 'dotenv/config'
+
+export default {
+  mainnet: {
+    miniWallet: {
+      initialOperatorThreshold: process.env.MINIWALLET_INITIAL_OPERATOR_THRESHOLD,
+      initialOperators: JSON.parse(process.env.MINIWALLET_INITIAL_OPERATORS || '[]'),
+      initialUserLimit: ethers.utils.parseEther(process.env.MINIWALLET_INIITIAL_USER_LIMIT || '1000000'),
```

Comment:
It's best to use primitive types in configs (e.g. string), since it cannot be assumed what other developers would use when they develop something on top of this package (e.g inheriting the config files, import it into a common config, etc.)